doc/man/lttng-load.1.txt

```diff
@@ -76,16 +76,22 @@ option:-i, option:--input-path='PATH'::
     or a file, instead of loading them from the default search
     directories.
 
-option:--override-url::
-    Provides the equivalent of the option:--set-url option
-    of the man:lttng-create(1) command. The validity of the url override is
-    dependent of the type of session(s) to be loaded. Refer to
-    man:lttng-create(1) for the accepted URL format. This option applies
```

Comment:
The mention of the "accepted" URL format seemed important to me. We accept a (weird) format that specifies two ports (data/ctrl).

Maybe it's just me and referring to the `--set-url` is sufficient.

What do you think?